### PR TITLE
Add web page for 2021 CF Workshop

### PR DESCRIPTION
**Pull request by ethanrd**

### CODE REVIEW

**Review by erget — APPROVED** [review]

File: Meetings/2021-Workshop.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+## Agenda (proposed - times approximate)
+
+### Tuesday, 21 September 2021
+**15:00 UTC - Welcome and Introductions (?)**
+
+**15:15 - Review of CF-1.9 (David Hassell)**
+
+**15:45 - Governance (?)**
+
+**16:15 - CF Standard Names (Alison Pamment)**
```
Comment (davidhassell) [inline Meetings/2021-Workshop.md:28]:
```suggestion
**16:15 - CF Standard Names (Alison Pamment, Sadie Bartholomew)**
```

File: Meetings/2021-Workshop.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+**15:30 - CF and the Global Community (Gui Castelao, Kevin O'Brien)**
+* OG 1.0 - OceanGliders data format (Gui)
+* Integraton of CF-netCDF profiles with WMO data distribution process (Kevin)
+
+**16:00 - Breakouts for Current Discussion Topics**
+
+**18:00 - Adjourn to Virtual Social Hour**
+* Virtual Social Hour
+
+### Wednesday, 22 September 2021
```
Comment (davidhassell) [inline Meetings/2021-Workshop.md:53]:
```suggestion
### Thursday, 23 September 2021
```

File: Meetings/2021-Workshop.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+**17:30 - Compression (Charlie Zender)**
+* Metadata to encode precision of lossy compression (Charlie Zender)
+* Lossy compression by coordinate sampling (Anders Sorensen)
+
+**18:00 - Adjourn to Virtual Social Hour**
+* Virtual Social Hour
+
+### Wednesday, 22 September 2021
+**15:00 UTC - Experiences with CMIP6 (Martin Juckes)**
```
Comment (davidhassell) [inline Meetings/2021-Workshop.md:42]: Did we also have Climate indices (Lars Baring, 1/2 hour) at the beginning of Day 2?

File: Meetings/2021-Workshop.md (hunk 1 above)
Comment (ethanrd) [reply Meetings/2021-Workshop.md:28]: Thanks @davidhassell ! I will fold this in.

File: Meetings/2021-Workshop.md (hunk 2 above)
Comment (ethanrd) [reply Meetings/2021-Workshop.md:53]: Oops! Thanks again.

File: Meetings/2021-Workshop.md (hunk 3 above)
Comment (ethanrd) [reply Meetings/2021-Workshop.md:42]: Yes. Good catch. I'll get that added.

**Comment by czender** [comment]
Hi Ethan,
Nice job. I suggest the name of the session I am coordinating be changed from "Compression" to "Lossy Compression", since I think both speakers will discuss CF issues related exclusively to lossy compression.
Charlie

**Comment by davidhassell** [comment]
Hi Ethan - thanks for putting this together - all looks good to me.
David

**Comment by ethanrd** [comment]
Thanks @czender - I just added "Lossy" to your session title.

**Comment by larsbarring** [comment]
Hi Ethan,
Looks good -- one small comment though regarding Wednesday 15:00; my family name is Bärring (ideally with the two dots, but without will be fine as well).
Lars

**Comment by davidhassell** [comment]
My Apologies, Lars - the mistake was mine.

**Comment by kevin-obrien** [comment]
Thanks Ethan - looks good to me

**Comment by ethanrd** [comment]
Sorry @larsbarring - I have made that change and will push shortly.

All - Thanks for the reviews. I think I've captured all the changes/suggestions. Once I push this latest change, I will merge (later today) if that sounds good to everyone.

**Pull request merged**